File: PULSE.Model/BicyclePriceStats.cs
namespace PULSE.Model
{
  public enum BicyclePriceStatsRange
  {
    TO_500,
    FROM_500_TO_1000,
    FROM_1000_TO_1500,
    FROM_1500_TO_2500,
    FROM_2500_TO_5000,
    FROM_5000
  }

  public partial class BicyclePriceStats
  {
    public BicyclePriceStatsRange? Range { get; set; }
    public decimal? Value { get; set; }
  }
}

File: pulse_admin/lib/api/types/stats/bicycle_price_stats.dart
import 'package:json_annotation/json_annotation.dart';

part 'bicycle_price_stats.g.dart';

enum BicyclePriceStatsRange {
  to500,
  from500to1000,
  from1000to1500,
  from1500to2500,
  from2500to5000,
  from5000
}

@JsonSerializable()
class BicyclePriceStats {
  int? range;
  double? value;

  BicyclePriceStats();

  factory BicyclePriceStats.fromJson(Map<String, dynamic> json) =>
      _$BicyclePriceStatsFromJson(json);

  Map<String, dynamic> toJson() => _$BicyclePriceStatsToJson(this);
}
